tests/nfv/prepare_env.pm

```diff
@@ -43,48 +43,50 @@ sub run {
     my ($self)         = @_;
     my $vsperf_repo    = "https://gerrit.opnfv.org/gerrit/vswitchperf";
     my $vsperf_version = get_required_var('VSPERF_VERSION');
-    my $dpdk_repo      = "http://dpdk.org/git/dpdk";
     my $vnf_image      = get_required_var('VNF_IMAGE');
     my $trafficgen_ip  = get_trafficgen_ip();
     my $children       = get_children();
     my $child_id       = (keys %$children)[0];
 
     record_info("INFO", "Install needed packages for NFV tests: OVS, DPKD, QEMU");
-    zypper_call('--quiet in git-core openvswitch-switch dpdk qemu tcpdump', timeout => 200);
+    zypper_call('--quiet in git-core openvswitch-switch dpdk qemu tcpdump', timeout => 60 * 4);
 
     assert_script_run("cd /root/");
     record_info("INFO", "Clone VSPerf repository");
-    assert_script_run("git clone --quiet --depth 1 --branch $vsperf_version $vsperf_repo", timeout => 200);
-    record_info("INFO", "Clone DPKD repository");
-    assert_script_run("git clone --quiet --depth 1 $dpdk_repo", timeout => 600);
+    assert_script_run("git clone --quiet --depth 1 --branch $vsperf_version $vsperf_repo", timeout => 60 * 4);
+
 
     record_info("INFO", "Start openvswitch service");
-    systemctl 'enable openvswitch', timeout => 200;
-    systemctl 'start openvswitch',  timeout => 200;
+    systemctl 'enable openvswitch', timeout => 60 * 2;
+    systemctl 'start openvswitch',  timeout => 60 * 2;
 
     record_info("INFO", "VSPerf Installation");
     assert_script_run("cd vswitchperf/systems");
+
+    # Hack to skip the OVS, DPDK and QEMU compilation as we will use the vanilla packages
+    assert_script_run("sed -n -e :a -e '1,8!{P;N;D;};N;ba' -i build_base_machine.sh");
     if (is_sle('>=15')) {
-        # Hack to skip the OVS, DPDK and QEMU compilation as we will use the vanilla packages
-        assert_script_run("sed -n -e :a -e '1,8!{P;N;D;};N;ba' -i build_base_machine.sh");
+        #zypper_call('--quiet in python2-pip');
```

Comment:
Please don't leave code commented out - noone will understand by reading the code thy this is commented..

Reply:
right, I forgot to remove it. Thanks for noticing